packages/core/types/index.js

```diff
@@ -505,8 +504,9 @@ export interface MutableAsset extends BaseAsset {
   type: string;
 
   addDependency(dep: DependencyOptions): string;
-  addIncludedFile(file: File): void;
+  addIncludedFile(filePath: FilePath): void;
```

Comment:
Debating renaming this to `invalidateOnFileChange` or something. I guess it's different from `invalidateOnFileUpdate` because it also invalidates on delete, and I don't think we need to make plugin authors specify both manually. Thoughts?

Reply:
Yeah I could see a function that sets up both invalidations being useful. Not sure if "change" totally communicates that, I could see people getting "update" and "change" mixed up, but I can't really think of a better word, so I think I'm okay with it :)

Reply:
Ok, let's add this to our API audit separately from this PR